editions/tw5.com/tiddlywiki.info

```diff
@@ -5,6 +5,7 @@
 		"tiddlywiki/browser-sniff",
 		"tiddlywiki/railroad",
 		"tiddlywiki/evernote",
+		"tiddlywiki/internals"
```

Comment:
missing comma at the end -  please update.

Reply:
Thanks @pmario